nifi-registry/nifi-registry-core/nifi-registry-flow-diff/src/main/java/org/apache/nifi/registry/flow/diff/StandardFlowComparator.java

```diff
@@ -129,7 +129,7 @@ private boolean compareComponents(final VersionedComponent componentA, final Ver
     }
 
     private boolean compareComponents(final VersionedComponent componentA, final VersionedComponent componentB, final Set<FlowDifference> differences,
-        final boolean compareName, final boolean comparePos, final boolean compareComments) {
+                                      final boolean compareName, final boolean comparePos, final boolean compareComments) {
```

Comment:
The changes in this file appear to be limited to formatting, can you revert the changes?

Reply:
Will revert the changes